deploy/single

```diff
@@ -0,0 +1 @@
+single-v1
```

Comment:
Do you mind explain where does the following single-v2 come from ?

Reply:
Is this a question about the layout or the contents? Manifests under single-v1 are generated by `hack/build-single-manifests.sh`. Manifests under single-v2 are generated by `railgun/hack/deploy/build-single-manifests.sh`.

The layout is from my understanding of Michal's request this morning, that we place both the v1 and v2 manifests in the same location, which I thought was reasonable. The addition of the `single -> single-v1` symlink was to ensure that existing references elsewhere do not break (we have a number of bit.ly links in documentation that point to the existing locations under `single`).

The Github UI display for symlinks is weird--it sorta looks like this is instead a file with a single line of content--but this is indeed a symlink.

Reply:
Resolved during a call. Github symlink display is just confusing :shrug: :shrimp: